src/client/pages/OfferNew/index.tsx

```diff
@@ -80,9 +78,7 @@ export const OfferNew: React.FC = () => {
 
   const handleCheckoutToggle = (open: boolean) => {
     toggleCheckout(open)
-    if (isMobile) {
-      Intercom('update', { hide_default_launcher: open })
-    }
+    Intercom('update', { hide_default_launcher: open })
```

Comment:
I think you have a point with intercom being accessible from the checkout but unfortunately it's then placed on top of the sign button - since the button is now almost as wide as the sidebar. It looks really weird imo. But maybe we can fins a solution for it that looks better.

Reply:
OK so I guess we're going with just removing the Intercom button for now, since Mange said that's fine? @gustaveen